indico/modules/rb/forms/rooms.py

```diff
@@ -115,6 +115,7 @@ class RoomForm(IndicoForm):
                                             [Optional(), NumberRange(min=1, max=9)], default=1)
     notification_for_responsible = BooleanField(_(u'Remind room manager too'))
     notifications_enabled = BooleanField(_(u'Reminders enabled'), default=True)
+    booking_limit_days = IntegerField(_(u'Booking limit in days'), [Optional(), NumberRange(min=1)])
```

Comment:
What about "Maximum length of booking (days)"? I think it better conveys what this is about.